css/terminal7.css

```diff
@@ -767,11 +749,20 @@ audio { display: none; }
     text-align: left;
     padding-left: 1.5em;
     text-shadow: none;
+    z-index: 20;
```

Comment:
It seems that gate-stats is also used in the map and it changes it.  You should move your changes to `.gate > .gate-stats` bellow
![image](https://github.com/tuzig/terminal7/assets/36852/0bed7df7-25ba-48b0-8421-2d63b7594bf5)